arm-machinelearning/2016-05-01-preview/swagger/commitmentPlans.json

```diff
@@ -0,0 +1,903 @@
+                },
+                "tier": {
+                    "type": "string",
+                    "description": "The SKU tier. Along with name, uniquely identifies the SKU."
+                }
+            }
+        },
+        "PaginatedResult[CommitmentAssociation]": {
+            "type": "object",
+            "description": "A paginated set of commitment association resources.",
+            "properties": {
+                "nextLink": {
+                    "type": "string",
+                    "readOnly": true,
+                    "description": "A URI to retrieve the next page of results."
+                },
+                "value": {
+                    "type": "array",
+                    "description": "The set of results for this page.",
+                    "items": {
+                        "$ref": "#/definitions/CommitmentAssociation"
+                    },
+                    "readOnly": true
+                }
+            }
+        },
+        "MoveCommitmentAssociationRequest": {
+            "type": "object",
+            "description": "Specifies the destination commitment plan for a move operation.",
+            "properties": {
+                "destinationPlanId": {
+                    "type": "string",
+                    "description": "The ARM ID of the commitment plan to re-parent the commitment association to."
+                }
+            }
+        },
+        "CommitmentPlan": {
+            "type": "object",
+            "description": "A commitment plan resource.",
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ],
+            "properties": {
+                "etag": {
+                    "type": "string",
+                    "description": "An entity tag used to enforce optimistic concurrency."
+                },
+                "properties": {
+                    "$ref": "#/definitions/CommitmentPlanProperties"
+                },
+                "sku": {
+                    "$ref": "#/definitions/ResourceSku"
+                }
+            }
+        },
+        "CommitmentPlanProperties": {
+            "type": "object",
+            "description": "Properties of a commitment plan resource.",
+            "properties": {
+                "chargeForOverage": {
+                    "type": "boolean",
+                    "description": "Indicates whether usage beyond the commitment plan's included quantities will be charged."
+                },
+                "chargeForPlan": {
+                    "type": "boolean",
+                    "description": "Indicates whether the commitment plan will incur a daily charge."
+                },
+                "creationDate": {
+                    "format": "date-time",
+                    "description": "The date at which this commitment plan was created.",
```

Comment:
Please describe the format in the description whenever the service deals with date, duration, date-time formats. This gives the user an idea about what to expect or provide. In this scenario should this be in ISO8601 format?